src/popup/utils/index.js

```diff
@@ -2,5 +2,5 @@
 export const store = {
   scripts: [],
   commands: [],
-  domains: [],
+  domain: "",
```

Comment:
lint error: Strings must use singlequote

Reply:
Thanks @gera2ld, I will fix this!